### PR TITLE
Create hetal7.json

### PR DESCRIPTION
**Pull request by hetal7**
Fremont CA lat & lon

### CODE REVIEW

File: _pins/hetal7.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: hetal7
+latitude: 37.55091° N
+longitude: -121.9822° E
```
Comment (hetal7) [inline _pins/hetal7.json]: Good change

Comment (coolc23) [reply _pins/hetal7.json]: You are missing the +--- under longitude

**Comment by brianamarie** [comment]
@hetal7  There are still a few errors in your file that are keeping the tests from passing. [Click "Help Me Troubleshoot" for some specific ideas](https://services.github.com/on-demand/intro-to-github/edit-files-on-github), or [watch this video to get some help](https://vimeo.com/201011288). If you need or want help, let me know! If this hasn't been resolved by the end of next week, I'll go ahead and make the changes for you so that it can be merged in. 😄

P.S. I think the problems may be:
- The latitude and longitude format is incorrect
- There is an extra line of `+---` at the end

**Comment by Davoudpur** [comment]
ok

**Pull request merged**